packages/card/README.md

```diff
@@ -25,6 +25,18 @@ yarn add @spectrum-web-components/card
 </sp-card>
 ```
 
+## Title
+
+The default application of a title for an `sp-card` is to leverage the `title` attribute. When HTML based title content is desired, the `title` slot is also available for deliverin this content.
```

Comment:
```suggestion
By default, the title for an `sp-card` is applied via the `title` attribute, which is restricted to string content only. When HTML content is desired, a slot named `title` available for applying the title.
```

English is the most difficult language we work in...

Reply:
lol, sounds good to me. done and done!